config.toml

```diff
@@ -145,7 +145,7 @@ fullversion = "v1.23.6"
 version = "v1.23"
 githubbranch = "main"
 docsbranch = "main"
-deprecated = false
+deprecated = true
```

Comment:
Any reference on the implications of this parameter?

Reply:
@tengqm `deprecated = true` is what triggers the deprecation banner for docs in previous releases: "Kubernetes v1.<stale> documentation is no longer actively maintained."

https://deploy-preview-34295--k8s-v1-23.netlify.app/docs/home/

Reply:
I see. "deprecated" in this context means "no longer actively maintained". That makes sense.

Reply:
I was surprised by the use of word "deprecated"/"deprecation" and I now realize it is not a big deal because we know what we are referring to.